README.md

```diff
@@ -101,6 +101,53 @@ scripts.
 If you would like to contribute your own demo, see our [demo submission
 guide](https://pennylane.ai/qml/demos_submission.html).
 
+## Build using Docker
+
+**Docker**  support exists for building using **CPU** and **GPU** (Nvidia CUDA 11.1+) images.
+
+**Note:** Docker builds using "make" will work on Linux and MAC only, For MS Windows one can use [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10)  
+
+### Build Basic Pennylane using Docker
+- To build a basic PennyLane image without any additional interfaces (Torch, TensorFlow, or Jax) and **plugins** (qiskit, amazon-bracket, cirq, rigetti) run following:

```

Comment:
The `and` should probably be changed to an `or` here: `...without any additional interfaces or plugins...`. Otherwise I think this statement would be slightly confusing/misleading.